go.mod

```diff
@@ -19,112 +19,112 @@ replace cuelang.org/go => github.com/sdboyer/cue v0.5.0-beta.2.0.20230419165817-
+	github.com/Masterminds/semver v1.5.0 // @grafana/backend-platform
+	github.com/VividCortex/mysqlerr v0.0.0-20170204212430-6c6b55f8796f // @grafana/backend-platform
+	github.com/aws/aws-sdk-go v1.44.234 // @grafana/aws-datasources
+	github.com/beevik/etree v1.2.0 // @grafana/backend-platform
+	github.com/benbjohnson/clock v1.3.3 // @grafana/alerting-squad-backend
+	github.com/bradfitz/gomemcache v0.0.0-20190913173617-a41fca850d0b // @grafana/backend-platform
+	github.com/centrifugal/centrifuge v0.29.1 // @grafana/grafana-app-platform-squad
+	github.com/crewjam/saml v0.4.13 // @grafana/backend-platform
+	github.com/fatih/color v1.15.0 // @grafana/backend-platform
+	github.com/gchaincl/sqlhooks v1.3.0 // @grafana/backend-platform
+	github.com/go-git/go-git/v5 v5.4.2 // @grafana/grafana-app-platform-squad
+	github.com/go-ldap/ldap/v3 v3.4.4 // @grafana/grafana-authnz-team
+	github.com/go-openapi/strfmt v0.21.7 // @grafana/alerting-squad-backend
+	github.com/go-redis/redis/v8 v8.11.5 // @grafana/backend-platform
+	github.com/go-sourcemap/sourcemap v2.1.3+incompatible // @grafana/backend-platform
+	github.com/go-sql-driver/mysql v1.7.1 // @grafana/backend-platform
+	github.com/go-stack/stack v1.8.1 // @grafana/backend-platform
+	github.com/gobwas/glob v0.2.3 // @grafana/backend-platform
 	github.com/gofrs/uuid v4.4.0+incompatible // indirect
-	github.com/gogo/protobuf v1.3.2
-	github.com/golang/mock v1.6.0
-	github.com/golang/snappy v0.0.4
-	github.com/google/go-cmp v0.5.9
-	github.com/google/uuid v1.3.0
-	github.com/google/wire v0.5.0
-	github.com/gorilla/websocket v1.5.0
-	github.com/grafana/alerting v0.0.0-20230606080147-55b8d71c7890
-	github.com/grafana/cuetsy v0.1.9
-	github.com/grafana/grafana-aws-sdk v0.15.0
-	github.com/grafana/grafana-azure-sdk-go v1.7.0
-	github.com/grafana/grafana-plugin-sdk-go v0.165.0
-	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
-	github.com/hashicorp/go-hclog v1.5.0
-	github.com/hashicorp/go-plugin v1.4.9
-	github.com/hashicorp/go-version v1.3.0
-	github.com/influxdata/influxdb-client-go/v2 v2.6.0
-	github.com/influxdata/line-protocol v0.0.0-20210311194329-9aa0e372d097
-	github.com/jmespath/go-jmespath v0.4.0
-	github.com/json-iterator/go v1.1.12
-	github.com/jung-kurt/gofpdf v1.16.2
-	github.com/lib/pq v1.10.6
-	github.com/linkedin/goavro/v2 v2.10.0
-	github.com/m3db/prometheus_remote_client_golang v0.4.4
-	github.com/magefile/mage v1.14.0
-	github.com/mattn/go-isatty v0.0.18
-	github.com/mattn/go-sqlite3 v1.14.16
-	github.com/matttproud/golang_protobuf_extensions v1.0.4
-	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
+	github.com/gogo/protobuf v1.3.2 // @grafana/alerting-squad-backend
+	github.com/golang/mock v1.6.0 // @grafana/alerting-squad-backend
+	github.com/golang/snappy v0.0.4 // @grafana/alerting-squad-backend
+	github.com/google/go-cmp v0.5.9 // @grafana/alerting-squad-backend
+	github.com/google/uuid v1.3.0 // @grafana/alerting-squad-backend
```

Comment:
same as above: I think it's used by a number of other services
```
go mod why github.com/google/uuid 
# github.com/google/uuid
github.com/grafana/grafana/pkg/infra/slugify
github.com/google/uuid
```

Reply:
thanks! i updated to BEP